torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -7853,6 +7853,74 @@ TEST_F(NVFuserTest, FusionReductionScheduler_CUDA) {
       lparams);
 }
 
+// This test checks if our system could correctly handles the case where both
+// reduction and trivial reduction exist in the fusion. Trivial reduction
+// deserve testing because trivial reduction is handled more like a broadcasting
+// rather than a reduction.
+TEST_F(NVFuserTest, FusionReductionWithTrivial_CUDA) {
```

Comment:
nit: naming could be better with something like `FusionReductionWithTrivialReduction` as the concept of "trivial" in nvFuser is overloaded.

Reply:
renamed